tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -854,20 +854,25 @@ func computeIterations(run *v1alpha1.Run, tls *pipelineloopv1alpha1.PipelineLoop
 			}
 		}
 	}
-	if from != -1 || to != -1 {
-		if from == -1 || to == -1 {
-			return 0, iterationElements, fmt.Errorf("The from or to parameters was not found in runs")
+	if from != to {
```

Comment:
do we handle `from == to` somewhere?

Reply:
from == to is not an error at the moment. Do you think it should be?
Actually, if it is 0, it means it is unset.

Reply:
from=0, to=0, step=1 would not be handled by the if condition here, and how will the controller handle it?

Reply:
It is an unset state. i.e. loop is not numeric range.

Reply:
I guess you didn't get my point. ha. the from=0, to=0 could be set in yaml. and how about from=1, to=1, step=1.
my point is those are possible values that you can set from yaml. as long as the controller can handle it well, then it would be good. I just point out the condition here doesn't cover those cases.

Reply:
We don't throw error in that case. What do you expect to happen?

Reply:
The case `from == to`, is handled by controller by not running the loop, I have confirmed this behaviour. This behaviour is unchanged, there is no error reported either. By handling that case my guess was, you are suggesting some error reporting to user.

Reply:
> The case from == to, is handled by controller by not running the loop, I have confirmed this behaviour. This behaviour is unchanged, there is no error reported either.


Thanks, this answers my question. based on the requirement from the issue. no error is expected for this case. If you can add some test cases which address the cases listed in the issue, that would be great.